Utilities/ShimUtils/sbat-info.tool

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# sbat-info.tool - Dump SBAT information from .efi executable, with additional
+# version and SBAT enforcement level information if the executable is Shim.
+#
+# Copyright (c) 2023, Michael Beaton. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+LIGHT_GREEN='\033[1;32m'
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+# Note: binutils can be placed last on path in macOS, to provide objcopy but avoid overwriting native tools.
+command -v objcopy >/dev/null 2>&1 || { echo >&2 "objcopy not found - please install binutils package."; exit 1; }
```

Comment:
Consider adding prefix env variable, e.g. `BINUTILS_PREFIX`, in my opinion. It is common to install GNU toolchain under a full name, e.g. `x86_64-gnu-linux-objcopy`, here `x86_64-gnu-linux-` being a prefix. This avoids conflicts with native utilities.

Reply:
I honestly wonder why not to use pyelftools, could have made things much easier. But that's a separate question from the above.

Reply:
Well partly because gathering up the motivation to write these tools in Python might have taken ... longer ...

Reply:
Okay, have implemented this. Thanks.